packages/generator-clay/app/templates/_README.md

```diff
@@ -0,0 +1,26 @@
+# <%= kebabCaseName %>
+
+Metal <%= componentName %> component
```

Comment:
I think we can use `repoDescription` here since we ask what the description is or we can leave a description pattern with just the name of the component and remove the question. what do you think ?